packages/bot-web-ui/src/components/chart/chart.tsx

```diff
@@ -21,6 +22,7 @@ interface TChartProps {
     wsSendRequest: (req) => void;
     wsSubscribe: (req, callback) => void;
     getMarketsOrder: (active_symbols) => void;
+    is_drawer_open: boolean;
```

Comment:
Isn't it better to sort the properties by alphabet?

Reply:
ordered alphabetically ✅